await myPromise
    // .then((result) => result + 'passando pelo then'); // then serÃ¡ executada caso for bem sucedida a promise 
    // .then((result) => result + 'ultimo then e ok'); 
    // .catch((error) => console.log(error.message)); // catch captura o erro e exige com console.log 